h2/src/main/org/h2/expression/analysis/DataAnalysisOperation.java

```diff
@@ -44,14 +44,31 @@
      */
     public static final int STAGE_WINDOW = 2;
 
+    /**
+     * SELECT
+     */
     protected final Select select;
 
+    /**
+     * Window clause / OVER
```

Comment:
It's actually an `OVER` clause.

`WINDOW` clause is a different story, it belongs to the `SELECT` command.

They both use `Window` class, however.